runtimes/common/src/constants.rs

```diff
@@ -164,11 +164,10 @@ pub mod bonded_coins {
 	/// The size is checked in the runtime by a test.
 	pub const MAX_POOL_BYTE_LENGTH: u32 = 970;
 	pub const BASE_DEPOSIT: Balance = deposit(1, MAX_POOL_BYTE_LENGTH);
-	/// 1 Storage entry in the assets pallet.
-	/// AssetId has 8 bytes. Two times stored in the storage. -> 16 bytes
-	/// struct AssetMetadata has 18 bytes + name + symbol -> 18 + 8 = 26 bytes
+	pub const ASSET_ID_BYTE_LENGTH: u32 = 16;
```

Comment:
They do not need to be `pub`

Reply:
removed all unnecessary pubs in: [5f97947](https://github.com/KILTprotocol/kilt-node/pull/818/commits/5f979479d8831bbf3992dfaa7476a7e96032e38a)